ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/page.tsx

```diff
@@ -38,6 +40,34 @@ export default function Page() {
         icon={<Squares2X2Icon className="h-5 w-5 text-white" />}
         links={navLinks}
         title="Apps"
+        action={
+          <HoverCardRoot>
+            <HoverCardTrigger asChild>
+              {/* To do: wire this to the docs */}
+              <NextLink
+                className="flex cursor-pointer items-center gap-1 rounded-md border border-slate-200 px-2 py-1 text-slate-200 hover:border-white hover:text-white"
+                href="/"
+                target="_blank"
+                rel="noreferrer noopener"
+              >
+                <InformationCircleIcon className="h-5 w-5" />
+                What are Apps?
+              </NextLink>
+            </HoverCardTrigger>
+            <HoverCardContent className="w-72 p-2.5 text-sm">
+              <p>
+                Each endpoint where you serve functions is an app. Each time you update your
+                functions, you must sync your app to Inngest. When using our Vercel or Netlify
+                Integrations, your app will be synced automatically.
+              </p>
+              <br />
+              <p>Deploys have been renamed to “Syncs.” Syncs are found within Apps.</p>
```

Comment:
```suggestion
              <p>Deploys have been renamed to “syncs.” Syncs are found within Apps.</p>
```

Nit: "sync" isn't a proper noun